src/Build/Resources/Strings.resx

```diff
@@ -1324,6 +1324,9 @@
     <value>Could not resolve SDK "{0}". Exactly one of the probing messages below indicates why we could not resolve the SDK. Investigate and resolve that message to correctly specify the SDK.
   {1}</value>
   </data>
+  <data name="SucceededToResolveSDK" xml:space="preserve">
+    <value>The SDK "{0}" was successfully resolved by the SDK resolver "{1}". SDK Result: "Path : {2}, Version : {3}"</value>
```

Comment:
Message wording nitpick: what about something like

```
The SDK "{0}" was successfully resolved by the "{1}" resolver to location "{2}" and version "{3}".
```

Other than that, love to see this!